src/gql/queries/dataset.ts

```diff
@@ -167,7 +200,7 @@ export const SYNC_DATASET = gql`
 `;
 
 export const IMPORT_GOOGLE_SHEET_DATASET = gql`
-  mutation importGoogleSheetDataset(
+  mutation ImportGoogleSheetDataset(
```

Comment:
Can you make these the same as the backend's importDatasetFromGoogleSheet? If we ever add more import dataset types it'd be nice to have
ImportDatasetFromGoogleSheet
ImportDatasetFromXXXXX
ImportDatasetFromYYYYY

Reply:
updated to ImportDatasetFromGoogleSheet